### PR TITLE
Add ability to dismiss attachment modal via swipe

### PR DESCRIPTION
**Pull request by Jag96**
@marcaaron will you please review this?

### Details
This PR adds the ability to swipe down and right to dismiss an attachment on mobile.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/154788

### Tests
1. Sign into an account and send an image attachment
2. After the attachment uploads, tap on it, confirm an attachment modal shows
3. Double tap on the image, confirm it zooms
4. Double-tap again, confirm it zooms out
5. Use two fingers to zoom in, confirm the image zooms in
6. While zoomed in, swipe down and right and confirm the modal doesn't close, and that the viewport pans around the zoomed image
7. Double-tap again then swipe down or right, confirm the modal dismisses
8. Confirm that there are no regressions on web
    1. Confirm attachments still show up on web
    2. Confirm the modal attachment closes normally on web

### QA Steps
Same as above

### Tested On
- [x] Web
- [x] Mobile Web
- [ ] Desktop
- [x] iOS
- [x] Android

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -402,6 +402,8 @@ PODS:
     - React-Core
   - RNCAsyncStorage (1.12.1):
     - React-Core
+  - RNCClipboard (1.5.1):
```
Comment (Jag96) [inline ios/Podfile.lock:405]: Looks like this wasn't updated previously, doing a `pod install` added these.

**Review by marcaaron — APPROVED** [review]
LGTM!

**Comment by isagoico** [comment]
Android/iOS - Chat - Image does not zoom in when two fingers are used

## Expected Result
Confirm the image zooms in

## Actual Result
Image does not zoom in

## Action Performed
1. Launch the app and login
2. Send an image attachment
3. After the attachment uploads, tap on it, confirm an attachment modal shows
4. Use two fingers to zoom in

## Platform

iOS ✔️ 
Android ✔️ 

**Notes/Images/Videos**
Issue is **reproducible in production** but it's failing step 5 of the PR. Let me know if I should raise this as a separate issue. 

https://user-images.githubusercontent.com/44479856/115468224-c4c6e180-a200-11eb-8a4e-9e751b56a07f.mp4

**Comment by Jag96** [comment]
@isagoico if this is happening on production then let's create a new issue for it, feel free to tag me in it and I'll have a look

**Pull request merged**